lib/src/generator/templates.dart

```diff
@@ -21,9 +19,14 @@ import 'intl/messages_all.dart';
 
 class $className {
   $className();
-  
-  static $className? current;
-  
+
+  static $className? _current;
+
+  static $className get current {
+    assert(_current != null, 'No instance of $className was loaded. Try to call $className.load(locale) before accessing $className.current.');
```

Comment:
Maybe we should update this message to something like _'No instance of $className was loaded. Try to initialize $className delegate before accessing $className.current.'_.